components/eamxx/src/share/atm_process/atmosphere_process.hpp

```diff
@@ -662,6 +681,58 @@ add_invariant_check (const Args... args) {
   add_invariant_check(fpc);
 }
 
+#ifdef EAMXX_HAS_PYTHON
+inline bool AtmosphereProcess::
```

Comment:
Note: these have to be inline, and in the hpp file, due to how pybind11 linkage works. Putting them in a cpp file causes undefined references.